sensorprocess/lidarsensorprocess.go

```diff
@@ -32,13 +33,18 @@ func (config *Config) addLidarReadingInOnline(ctx context.Context) error {
 	// get next lidar data response; ignoring status since it is always false
 	lidarReading, err := config.Lidar.TimedLidarReading(ctx)
 	if err != nil {
+		if errors.Is(err, replaypcd.ErrEndOfDataset) {
+			time.Sleep(1 * time.Second)
+		}
 		return err
 	}
 
 	// add lidar data to cartographer and sleep remainder of time interval
 	timeToSleep := config.tryAddLidarReadingOnce(ctx, lidarReading)
-	time.Sleep(time.Duration(timeToSleep) * time.Millisecond)
-	config.Logger.Debugf("lidar sleep for %vms", timeToSleep)
+	if !lidarReading.TestIsReplaySensor {
```

Comment:
[q] I thought this sleep was being removed/ reduced to something like 10 ms?

Reply:
[note] used in localization mode and if a user forces use_cloud_slam to false during mapping. Was previous unused due to us treating online and offline identically

Reply:
*unused due to us treating replay sensors the same as live sensors